packages/language-server/src/codeActionProvider.ts

```diff
@@ -35,7 +35,7 @@ function getInsertRange(document: TextDocument): Range {
  *        (0.4 allows 1 substitution/transposition for every 5 characters,
  *         and 1 insertion/deletion at 3 characters)
  */
-function getSpellingSuggestionForModelAndEnumName(
+function getSpellingSuggestions(
   name: string,
   relations: string[],
```

Comment:
Probably change the arguments' names? They are no longer just relations but also keywords etc.